src/advanced/components/pages/admin/coupon/CouponCard.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Coupon } from '../../../../types';
+
+interface CouponCardProps {
+  coupon: Coupon;
+  onRemove: (code: string) => void;
+}
+
+export const CouponCard: React.FC<CouponCardProps> = ({ coupon, onRemove }) => {
+  return (
+    <div className='relative bg-gradient-to-r from-indigo-50 to-purple-50 rounded-lg p-4 border border-indigo-200'>
+      <div className='flex justify-between items-start'>
+        <div className='flex-1'>
+          <h3 className='font-semibold text-gray-900'>{coupon.name}</h3>
+          <p className='text-sm text-gray-600 mt-1 font-mono'>{coupon.code}</p>
+          <div className='mt-2'>
+            <span className='inline-flex items-center px-3 py-1 rounded-full text-sm font-medium bg-white text-indigo-700'>
+              {coupon.discountType === 'amount'
+                ? `${coupon.discountValue.toLocaleString()}원 할인`
+                : `${coupon.discountValue}% 할인`}
```

Comment:
저는 jsx 코드는 진짜 UI만 담당할 수 있도록 이런 부분도 최대한 변수로 분리하려고 하는 편인데요. 다른 분들 의견은 어떨지 궁금하네요

```suggestion
const discountLabel =
  coupon.discountType === 'amount'
    ? `${coupon.discountValue.toLocaleString()}원 할인`
    : `${coupon.discountValue}% 할인`;

<span className='inline-flex items-center px-3 py-1 rounded-full text-sm font-medium bg-white text-indigo-700'>
    {discountLabel}
<span>
```